src/sora-v3.md

```diff
@@ -84,9 +81,24 @@ However, this can be a costly way to obtain funds if a very large bonus has to b
 
 #### The Vision for SORA V3
 
-We can build a sustainable and self-reinforcing economy by using buy-back-and-burn mechanisms (a portion of the VAL and PSWAP re-mint to go to burning KUSD, while still keeping VAL and PSWAP strictly deflationary, as well as a portion of the XOR network fees paid with each transaction) to help KUSD maintain its peg, with a large TBCD treasury to cover any gaps, with an additional mechanism to buy-back-and-burn TBCD (10% of the remint of VAL and PSWAP—which captures both the use and volume of activity on the network). Through this process, we can create a powerful vortex that will eventually lead to the burning of a large amount of the XOR supply while still creating conditions for financing new output in the SORA economy. Yes, it’s a long-term play, but creating a new world economy that isn’t based on debt and enslavement will take time and have a never-ending share of sceptics and critics.
+We can build a sustainable and self-reinforcing economy by using buy-back-and-burn mechanisms. Through this process, we can create a powerful vortex that will eventually lead to the burning of a large amount of the XOR supply while still creating conditions for financing new output in the SORA economy, as documented below in the transaction fee breakdown.
 
-In the SORA economy, XOR is the predictable monetary base. VAL and PSWAP are on top of the XOR monetary base, incentivizing validators and liquidity providers. Finally, KUSD is on top of the pyramid, the all-seeing eye of the monetary system, reaching out into the existing dollarized world economy. Transaction fees propagate through the SORA economy, like blood pumping, burning some XOR, swapping through VAL and PSWAP, buying back, and burning TBCD and KUSD. Like a fine tapestry woven to perfection, each layer of the economy is thus encouraged to build value for the SORA economy by contributing their work and ideas so that by working together, the entire SORA ecosystem will flourish.
+Yes, it’s a long-term play, but creating a new world economy that isn’t based on debt and enslavement will take time and have a never-ending share of sceptics and critics.
+
+![](/.gitbook/assets/sora-v3-5.png)
+
+In the SORA economy, XOR is the predictable monetary base. VAL and PSWAP are on top of the XOR monetary base, incentivizing validators and liquidity providers. 
```

Comment:
PSWAP is specific to Polkaswap (SORA v2) (DEX token). On SORA v3, the branding will be different, so PSWAP will have an equivalent but under another name. If we want to stick to v3 information here, I suggest we remove the reference to PSWAP.

Reply:
Edited